lib/phoenix_html/form.ex

```diff
@@ -332,6 +332,10 @@ defmodule Phoenix.HTML.Form do
     do: "#{id}_#{field}"
   def input_id(name, field) when is_atom(name),
     do: "#{name}_#{field}"
+  def input_id(name, field, value) do
```

Comment:
I don't think we need to make this function public. What if we call it `radio_button_id` and keep it private for now?

Reply:
This is basically so I can do `label(name, field, for: input_id(name, field, value))`. Otherwise I gotta re-implement that again. Also this method is useful for checkbox collection (when you have `name: "search[keys][]` for example) and need unique ids just like for radio buttons.

Reply:
I see, agreed. If that's the case, we need to add documentation to it, as it is effectively a separate function than `input_id/2`.

Reply:
Will do